package.json

```diff
@@ -121,7 +121,7 @@
     "test:framework:react:run": "mocha ./test/framework/twilio-video-react.js",
     "test:framework:react": "npm-run-all test:framework:react:*",
     "test:framework:install": "npm install chromedriver && npm install selenium-webdriver && npm install geckodriver && npm install puppeteer",
-    "test:framework": "npm-run-all test:framework:install test:framework:angular test:framework:no-framework test:framework:react",
```

Comment:
Why are we removing angular tests?

Reply:
After spending some time with angular tests, I've added the work to address the angular test failures in a future ticket here: [JSDK-3110](https://issues.corp.twilio.com/browse/JSDK-3110). I think fixing the angular tests would be impossible unless we updated some of the packages which caused problems for other older packages as well.